src/marqo/core/models/marqo_index.py

```diff
@@ -188,6 +190,39 @@ def _update_model_properties_from_registry(self) -> None:
                 raise InvalidArgumentError(
                     f'Invalid model properties for model={model_name}. Reason: {e}.'
                 )
+            
+    def get_text_query_prefix(self, request_level_prefix: Optional[str] = None) -> str:
+        if request_level_prefix is not None:
+            return request_level_prefix
+
+        # For backwards compatibility. Since older versions of Marqo did not have a text_query_prefix field,
+        # we need to return an empty string if the model does not have a text_query_prefix. 
+        # We know that the value of text_query_prefix is None in old indexes since the model was not populated
+        # from the registry.
+        if self.text_query_prefix is None:
+            return ""
+
+        # Else return the model default as populated during initialization
+        return self.text_query_prefix or ""
```

Comment:
You've already covered None so just `return self.text_query_prefix` will suffice

Reply:
Removed